test/interface.jl

```diff
@@ -27,7 +27,7 @@
                 logp,
                 sample_per_iter;
                 max_iters=5_000,
-                optimiser=Optimisers.ADAM(0.01 * one(T)),
+                optimiser=Optimisers.Adam(0.01 * one(T)),
```

Comment:
Is this a deprecation or something? If so, should we maybe add compat entry to the tests?

Reply:
yeah, it is deprecated, see https://github.com/FluxML/Optimisers.jl/blob/6eaf26da61f3f1f7f6f663b0d413e3c124742014/src/rules.jl#L1C1-L1C2. I'll update the compat entry.